src/test/resources/schema/valid_robotics_agreed.json

```diff
@@ -10,7 +10,9 @@
     "phoneNumber": "07683736513",
     "postCode": "CM12 0HN",
     "title": "Mr",
-    "townOrCity": "Perferendis nemo id velit dolore qui officia"
+    "townOrCity": "Perferendis nemo id velit dolore qui officia",
+    "isAppointee": "No",
```

Comment:
Please explain what the point of this is? It makes no sense to me

Reply:
It's removed now. Thanks.